ts/nni_manager/training_service/reusable/trialDispatcher.ts

```diff
@@ -62,6 +71,8 @@ class TrialDispatcher implements TrainingService {
     private enableGpuScheduler: boolean = false;
     // uses to save if user like to reuse environment
     private reuseEnvironment: boolean = true;
+    private logCollection: string = '';
+    private environmentMaintenceLoopInterval: number = 5000;
```

Comment:
It's better to read it from all services, and take max one. It can improve performance on local and other lightweight services.

Reply:
updated.